packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.controller.js

```diff
@@ -322,4 +322,67 @@ export default class DedicatedServerDashboard {
       type: 'navigation',
     });
   }
+
+  onChangeIntervention(noIntervention) {
+    const { monitored } = this.server;
+    this.updatingNoIntervention = true;
+    this.Server.updateMonitoring(
+      this.$stateParams.productId,
+      monitored,
+      noIntervention,
+    )
+      .then(() => {
+        this.server.noIntervention = noIntervention;
+        if (noIntervention === true) {
+          this.Alerter.success(
+            this.$translate.instant(
+              'server_configuration_no_intervention_activate_success',
```

Comment:
If condition on `noIntervention` could be used only as ternary operator for translationKey